docs/governance-workbook/README.md

```diff
@@ -6,6 +6,125 @@
+-	Storage is listing Azure File Sync, Azure Backup, Storage Accounts, Key Vaults
+
+## Security
+The **Security** tab is focusing on the security score for your subscriptions and controls
+- Security Scores by Subscription
+- Security Scores by Control
+- Top 5 attacked resources (with High Severity)
+- Top alert types
+- New Alerts (Since last 24hrs)
+- MITRE ATT&CK tactics
+- Active Alerts
+
+## Monitoring
+The **Monitoring** tab is providing Service Health information and main events that are happening into one selected subscription:
+-	All Service Health active Incident
+-	All changes performed on your resources for the past one day
+-	All deleted resources for the past 14 days
+
+## Service retirement
+The **Services retirement** tab shows Azure services that are being phased out so that you can mitigate affected resources
+
+## Resource age
+The **Resource age** tab is giving you more information about the resource “Creation Date” and the “Last Change Date” in the selected Subscription to help you to identify old resources and perform sanitization.
+
+## Tag explorer
+The **Tag explorer** tab help you to filter/sort your resources by Tag. You can list and identify resources with or without a specified tag name and with or without a value. Each result can be filtered by resource type.
+You can also get general information on Subscription and Resource Groups.
+
+## Cost management
+The **Cost management** tab is providing you high level information about your cost and can be filtered by tag.
+
+## Usage + limits
+Many Azure services have quotas, which are the assigned number of resources for your Azure subscription. Each quota represents a specific countable resource, such as the number of virtual machines you can create, the number of storage accounts you can use concurrently, the number of networking resources you can consume, or the number of API calls to a particular service you can make.
+The **Usage & limits** tab is providing this information about your subscriptions. If you need more information about quotas, see [Quotas overview](https://learn.microsoft.com/en-us/azure/quotas/quotas-overview).
+
+## Compliance
+The **Compliance** tab allow you to monitore your policy compliance, the number of failures by resources, by operations and by category.
+
+## Governance
+Microsoft Defender for Cloud continuously assesses your hybrid and multi-cloud workloads and provides you with recommendations to harden your assets and enhance your security posture.
+Central security teams often experience challenges when driving the personnel within their organizations to implement recommendations. The organizations' security posture can suffer as a result.
+We're introducing a brand-new, built-in governance experience to set ownership and expected remediation timeframes to resolve recommendations.
+Pre-requisite: To use this governance report, you need to create security governance rules. 
```

Comment:
Should we consolidate this into a single set of setup instructions? We can do that outside of this PR. Just thinking out loud. You can look at the [hubs docs](https://microsoft.github.io/finops-toolkit/hubs#-create-a-new-hub) for an example. The goal should be a simple set of steps, not overly detailed. We don't want them to think they have to do too much to get up and running.
```suggestion

Pre-requisite: To use this governance report, you need to create security governance rules.

```